client/src/components/PredictionPanel.vue

```diff
@@ -1,16 +1,28 @@
 <template>
-  <div class='container'>
-    <div class='explain-text'>
-      {{ ScorecardSummaryMessages.LEADOUT }}
+  <div class='container-column'>
+    <div class='container-row justify-right'>
+      <map-geocoder-wrapper class='search' v-model:expandSearch='showSearch' />
     </div>
-    <map-geocoder-wrapper v-model:expandSearch='showSearch' />
-    <div class='h1-header semi-bold'
-         v-if='this.pwsId != null && percentLead != null'>
-      Your neighborhood water system {{ pwsId }} is likely
-      {{ percentLead }}% lead.
-    </div>
-    <div class='explain-text'>
-      {{ ScorecardSummaryMessages.LEAD_LIKELIHOOD_EXPLAINED }}
+    <div class='container-column center-container'>
+      <div class='explain-text'>
+        {{ ScorecardSummaryMessages.LEADOUT }}
+      </div>
+      <div class='h1-header semi-bold'
+           v-if='showWaterSystemPrediction'>
```

Comment:
silly q - where is parcel message?

Reply:
Added a variable "showParcelPrediction" for clarity